### DISCUSSION TITLE
Why are the Remnant not swimming in jump drives?

### DISCUSSION THREAD

**Discussion by tuuktuuk** [endless-sky/endless-sky#10882] [Lore]
The Remnant repeatedly tell that they are farming Korath ships for outfits. Doesn't that mean that the Remnant are sitting on probably hundreds of jump drives? Why are they not using them?

Is there some religious reason? Or is there something about the way the Remnant strip outfits off ships or in the way they install outfits that have made the Korath jump drives go spectacularly kaboom, obliterating whole cities/spaceships? Perhaps even star systems? If so, why is there no mention such as "we never touch their jump drives because _______"?

**Comment by TheGiraffe3** [comment]
@Zitchas would know more about this, but from my understanding:
- Many of the Korath ships the Remnant raid have broken Jump Drives; reverse engineering is being worked on but not yet complete.
- The Remnant do in fact have many Jump Drives, but there are still few enough that a loss would be harmful and so they don't use them often. Zitchas' Remnant person ship (the Heron) uses a Jump Drive.

**Comment by Zitchas** [comment]
The Remnant *do* have a lot of Jump Drives, but they are prefer to use scram drives where possible as those are more tolerant of movement and harder to lock down. The vast majority of ships that the player sees are local defense, resource collection, transport, or other activities within the Ember Waste. And those will be effectively all SD instead of JD. They've got JD waiting to throw onto their ships if they need to, (and transports with them equipped in most settlements) but they're basically the Remnant equivalent of escape pods, not daily use things.

There's also the fact that they want those Korath ships to *leave* again. Otherwise the Korath don't get the resources they came for, and start classifying the Remnant as "too hard a target, not worth it to raid", at which point their source of advanced Korath tech to study dries up. So the Remnant don't take every Jump Drive they could get their hands on. A lot of them, yes, (The Korath have the capability to build, repair, and outfit ships in space, at least in lore) but not all of them.

**Reply by tuuktuuk** [reply]
Good answer :)
I hope parts of this will eventually make their way to the Remnant storyline.

**Reply by Zitchas** [reply]
Thank you. 

It definitely will, eventually. I have volunteers of lore and backstory for them, it just requires the proper conditions and mindset to convert it into enjoyable story.

**Reply by Quotatron3000** [reply]
Could I suggest an additional reason?

Remnant space is filled with EM distortions that interfere with electronic systems. Hyperdrives are hardier and more reliable, and if navigation on that malfunctions, oh well. But if navigation on a jump drive malfunctions, well you could end up in the middle of deep space stranded.
